src/domain/event/EventUtils.ts

```diff
@@ -118,10 +116,7 @@ export const getEventPlaceholderImageUrl = (
   event: EventFieldsFragment
 ): string => {
   const numbers = event.id.match(/\d+/g);
```

Comment:
Super small thing but if number part of the id is needed somewhere else, this could be separate method like `const numbers = getNumberId(event);`

Reply:
Can't think of another place where this would be used right now so I'm not gonna abstract it at this point.